src/main/java/com/team766/hal/wpilib/WPIRobotProvider.java

```diff
@@ -174,7 +174,9 @@ public Clock getClock() {
 
 	@Override
 	public boolean hasNewDriverStationData() {
-		return DriverStation.isNewControlData();
+		// TODO: replace implementation with event counting one
+		// return DriverStation.isNewControlData();
+		return true;
```

Comment:
Unless we need 2023 support immediately in other repos, I'd prefer to wait to upstream this until we have the implementation for this